packages/cf-component-card/src/CardControl.js

```diff
@@ -3,16 +3,19 @@ const {PropTypes} = React;
 
 class CardControl extends React.Component {
   render() {
+    const className = 'cf-card__control' + (this.props.wide ? ' cf-card__control--wide' : '');
```

Comment:
Hey that's really cool! I think this should be addressed across the entire repo so I opened  issue #49